scrapy/resolver.py

```diff
@@ -16,8 +16,7 @@ def __init__(self, reactor, cache_size, timeout):
     def getHostByName(self, name, timeout=None):
         if name in dnscache:
             return defer.succeed(dnscache[name])
-        if not timeout:
-            timeout = self.timeout
+        timeout = (self.timeout,)
```

Comment:
@redapple could you please add a comment above this line which explains why `timeout` argument is ignored?